src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx

```diff
@@ -27,6 +33,9 @@ class ConfirmationPage extends React.Component {
   }
 
   componentDidMount() {
+    if (!_.get(this.props.form.data, 'appliedForVAEducationBenefits', true)) {
+      window.location.replace(url1990);
```

Comment:
Will the veteran be able to see the confirmation information with this behavior?

Reply:
Not if they answer the question on the page with "Have you already submitted an Application for VA Education Benefits (VA Form 22-1990)?" with the answer "No". 

I asked @Jennywalter if this behavior was ok and she said yes. 

The story is https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16450

Reply:
Changing this behavior, adding a button to Confirmation page to direct to start of 1990